lib/reverse_string.rb

```diff
@@ -1,4 +1,12 @@
 def reverse_characters(str)
+puts 'Please give me a sentance to mirror'
+string = gets.chomp
```

Comment:
This method is meant to be generic–it should work for **any** string at all. And that hypothetical string is already represented by `str` (the "parameter" of this method). So you can remove `string = gets.chomp`–just imagine you have a string already, and it's stored (already) in a variable named `str`.

Don't forget to update the rest of your code to make sure you're referring to `str` instead of `string`.

(You can remove the `puts` line too, but it should still work even if it's there.)